src/fs.rs

```diff
@@ -49,6 +48,18 @@ pub fn mkdir(path: &Path) -> std::io::Result<()> {
   })
 }
 
+pub fn rmdir(path: &Path) -> std::io::Result<()> {
+  debug!("rmdir {}", path.display());
+  std::fs::remove_dir_all(path).or_else(|err| {
+    // Ignore NotFound error
+    if err.kind() == std::io::ErrorKind::NotFound {
```

Comment:
I prefer match expressions for those. They are shorter and more idiomatic Rust.

match err.kind() {
    std::io::ErrorKind::NotFound => Ok(),
    _ => Err(err)
}

Reply:
Done, thanks!